core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -1658,17 +1661,17 @@ object Parser extends ParserInstances {
       str
     }
 
-    case class StringP[A](parser: Parser[A]) extends Parser[String] {
+    case class StringP[A](parser: Parser0[A]) extends Parser0[String] {
       override def parseMut(state: State): String =
-        Impl.string(parser, state)
+        Impl.string0(parser, state)
     }
 
-    case class StringP1[A](parser: Parser1[A]) extends Parser1[String] {
+    case class StringP1[A](parser: Parser[A]) extends Parser[String] {
       override def parseMut(state: State): String =
-        Impl.string(parser, state)
+        Impl.string0(parser, state)
     }
 
-    case object StartParser extends Parser[Unit] {
+    case object StartParser0 extends Parser0[Unit] {
```

Comment:
I don't think the name needs to change here, there is only one StartParser, similar for EndParser

Reply:
a find-and-replace mishap